icu4j/build.properties

```diff
@@ -8,5 +8,5 @@ api.report.version = 70
 api.report.prev.version = 69
 release.file.ver = 70rc
```

Comment:
For this one - we want to keep this to generate series of release binaries with file name including 'rc'. So this one is OK for RC. We'll need to change this one for GA. I think we want a better process.